test/integration/react-current-version/test/index.test.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-env jest */
+
+import { join } from 'path'
+
+import cheerio from 'cheerio'
```

Comment:
Maybe we can make it a e2e test and specify the version? And we can test both 18 and 19 beta?

Reply:
Are we planning on support 18 and 19 in Next.js?